docs/app/services/app-configuration/app-configuration.service.ts

```diff
@@ -43,6 +43,10 @@ export class AppConfiguration {
         return this.config['plunker'];
     }
 
+    get getUniversalURL(): string {
```

Comment:
```suggestion
    get getUniversalUrl(): string {
```
This is our capitalization guideline; see properties above for other examples.